packages/transport/src/index.ts

```diff
@@ -11,22 +6,38 @@ import WebUsbPlugin from './lowlevel/webusb';
 import * as protobuf from 'protobufjs/light';
 import * as Long from 'long';
 
+import * as COMMON_ERRORS from './errors';
+import * as SESSION_ERRORS from './sessions/errors';
+import * as TRANSPORT_ERRORS from './transports/errors';
+import * as INTERFACE_ERRORS from './interfaces/errors';
+
```

Comment:
I am not sure if having all those errors separated make sense, since there are not that many of them, maybe having them all in `./errors.ts` would be more handy.

But maybe makes sense, just wondering if you have a clear preference about it.

Reply:
I am not sure yet myself. lets keep this topic open. It felt like a good idea to have a distict separation of layers from which these errors may originate